bin/sequenceserver

```diff
@@ -381,6 +384,18 @@ MSG
         exit
       end
 
+      if download_taxdb?
+        url = 'ftp://ftp.ncbi.nlm.nih.gov/blast/db/taxdb.tar.gz'
+        archive = File.join('/tmp', File.basename(url))
+        cmd = "curl -ip4 -C - #{url} -o #{archive} && "    \
+                 'mkdir -p ~/.sequenceserver && '             \
```

Comment:
The exact same code is used for downloading pre-compiled BLAST+ binaries. I suggest you extract this into a method with `url` as the only parmeter.